packages/hash/frontend/src/components/layout/PageHeader/AccountDropdown.tsx

```diff
@@ -118,36 +129,36 @@ export const AccountDropdown: VoidFunctionComponent<AccountDropdownProps> = ({
         }}
         PaperProps={{
           elevation: 4,
-          sx: {
+          sx: ({ palette }) => ({
             width: 225,
             borderRadius: "6px",
             marginTop: 1,
-            border: `1px solid ${theme.palette.gray["20"]}`,
-          },
-        }}
-        sx={{
-          color: theme.palette.gray[40],
+            border: `1px solid ${palette.gray["20"]}`,
+          }),
         }}
+        sx={({ palette }) => ({
+          color: palette.gray[40],
+        })}
       >
         <Box px={2} pt={1} pb={1.5}>
           <Typography
             variant="smallTextLabels"
-            sx={{
-              color: theme.palette.gray[80],
+            sx={({ palette }) => ({
+              color: palette.gray[80],
               fontWeight: 700,
-            }}
+            })}
           >
             {user.properties.preferredName}
           </Typography>
           <Typography
             component="p"
             variant="microText"
-            sx={{ color: theme.palette.gray[60], lineHeight: 1 }}
+            sx={({ palette }) => ({ color: palette.gray[60], lineHeight: 1 })}
           >
-            @{user.properties.shortname!}
+            {userEmail}
           </Typography>
         </Box>
-        <Divider sx={{ borderColor: theme.palette.gray[30] }} />
+        <Divider sx={({ palette }) => ({ borderColor: palette.gray[30] })} />
         <Box>
           <Link noLinkStyle href="#" onClick={() => setOpen(false)}>
             <ListItemButton
```

Comment:
Hover state BG colour does not match designs:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/42802102/158066332-a83a2b85-0fb5-42c5-803c-8d580d2ee062.png">

Reply:
https://github.com/hashintel/hash/pull/397/commits/5f616c622330c7ecdd90a63c95e8ca3ced87f11c